CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 cmake_minimum_required(VERSION 3.2)
 
+enable_language(CXX)
+enable_language(C)
+if (NOT CMAKE_BUILD_TYPE)
```

Comment:
We're already doing this here:
https://github.com/Project-OSRM/osrm-backend/blob/06b1b980bbdfc838c4f8e8cb370cba6dfb0329b0/CMakeLists.txt#L230-L233